src/components/h5p-button.js

```diff
@@ -56,8 +56,34 @@ H5P.Components.Button = (function () {
       H5P.Tooltip(button, { text: params.tooltip, position: params.tooltipPosition ?? 'top' });
     }
 
+    if (params.icon && H5P.Components.ButtonIconOnlyObserver) {
+      H5P.Components.ButtonIconOnlyObserver.observe(button);
+    }
     return button;
   }
-
   return Button;
 })();
+
+H5P.Components.ButtonIconOnlyObserver = H5P.Components.ButtonIconOnlyObserver || new ResizeObserver(entries => {
```

Comment:
If you update the modules (the eslint config in particular), you'll note that H5P Group's rules set the maximum line size to 100. Just a note, because the whole formatting of the library should be fixed in a separate issue. Good to have that in mind though.

Also: Regardless of the number of arguments, they should be enclosed in parantheses in arrow functions. That's also one of the rules.